scripts/unsupported/combine_tracks_postprocessing_cnv/aggregate_combined_tracks.wdl

```diff
@@ -4,13 +4,20 @@ workflow AggregateCombinedTracksWorkflow {
     Array[File] tumor_with_germline_filtered_segs
     Array[File] normals_igv_compat
     Array[File] tumors_igv_compat
+    Array[File] tumors_gistic2_compat
 
     call TsvCat as TsvCatTumorGermlinePruned {
         input:
             input_files = tumor_with_germline_filtered_segs,
             id = group_id + "_TumorGermlinePruned"
     }
 
+    call TsvCatNoHeader as TsvCatTumorGermlinePrunedGistic2 {
```

Comment:
Pruned -> Filtered (here and elsewhere throughout this file, not just in commit) to be consistent with last PR?

Reply:
Done